packages/neuron-wallet/src/database/chain/entities/asset-account.ts

```diff
@@ -60,7 +60,7 @@ export default class AssetAccount {
       this.sudtTokenInfo.decimal,
       this.balance,
       this.blake160,
-      this.id
+      this.id ?? undefined
```

Comment:
Why could the `id` be null? We need a comment here. otherwise, this line of code might be removed by someone else in the future.

Reply:
I forget to remove it, I will remove it later.